src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs

```diff
@@ -2,26 +2,27 @@
 {
     using System;
     using System.Collections.Generic;
-    using System.Data.Entity;
     using System.Linq;
     using Contracts;
     using Data.Queries.Common;
     using Data.Queries.GameResult;
     using Domain.GamesAggregate;
     using Entities;
+    using System.Data.Entity;
 
+#pragma warning disable S1200 // Classes should not be coupled to too many other classes (Single Responsibility Principle)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)